definitions/checks/sca_only.rb

```diff
@@ -0,0 +1,57 @@
+class Checks::CheckOrganizationContentAccessMode < ForemanMaintain::Check
+  metadata do
+    label :check_organization_content_access_mode
+    description 'Check if any organizations are using entitlement mode'
+    tags :pre_upgrade
+
+    confine do
+      feature(:katello)
+    end
+  end
+
+  def run
+    if feature(:instance).downstream
+      versioned_instance_name = "Satellite 6.16"
+      doc_link = "https://access.redhat.com/articles/simple-content-access"
+    else
+      versioned_instance_name = "Katello 4.12"
+      doc_link = "https://theforeman.org/plugins/katello/nightly/user_guide/simple_content_access/index.html"
```

Comment:
Katello 4.12 docs weren't available yet when I originally wrote this, so I pointed it at nightly as a temporary measure

But maybe it should point at 4.11 docs instead? I'm not sure if the docs will continue to document a distinction that no longer exists.

Reply:
Indeed, 4.12 docs will have all SCA-related info removed. 4.11 docs is a better bet I'd say.

Reply:
Actually docs.theforeman.org would be the more correct upstream documentation in general (the link you had is for the old doc site.) But docs.theforeman.org doesn't really have any good overview of SCA. I'm actually thinking maybe just use https://access.redhat.com/articles/transition_of_subscription_services_to_the_hybrid_cloud_console

for both upstream and downstream. This is what we also ended up using in https://bugzilla.redhat.com/show_bug.cgi?id=2255222. Thoughts?

Reply:
...and SCA is only relevant for folks using Red Hat content to begin with, so they'd already have an account

cool, I'll just link that one in either case.

Reply:
Oh, and it looks like this was recently updated, too! So this is the link to use. https://access.redhat.com/articles/transition_of_subscription_services_to_the_hybrid_cloud_console#satellite-13

Reply:
The absence of an answer for disconnected users here is concerning to me:

```
Satellite

Question: What if I do not want to use the Simple Content Access?
Answer: Satellite versions <= 6.15 are the only versions that support entitlement mode. Eventually, as per the [Red Hat Satellite Lifecycle](https://access.redhat.com/support/policy/updates/satellite) document, those versions will be out of support eventually.

Question: As a satellite user, how do I track my usage of subscriptions?
Answer: Install and/or enable the [Satellite inventory upload plugin](https://access.redhat.com/documentation/en-us/subscription_central/1-latest/html-single/getting_started_with_the_subscriptions_service/index#proc-installing-satellite-inventory-upload-plugin_assembly-setting-up-subscriptionwatch-ctxt)

Question: I am a disconnected user of Satellite and I cannot send usage data to console.redhat.com?
Answer:

Question: The manifest UI in console.redhat.com doesn't have any way to attach subscriptions. How do I attach subscriptions to a manifest?
Answer: The preferred pattern for a connected Satellite customer is to download the manfiest from console.redhat.com, import it, and add subscriptions via the Satellite user interface. [Learn more about creating manifests for a connected Satellite Server](https://access.redhat.com/documentation/en-us/subscription_central/2023/html/creating_and_managing_manifests_for_a_connected_satellite_server/index). If you are a disconnected Satellite or Ansible Automation Platform user, please continue to use the [Subscription Allocations](https://access.redhat.com/management/subscription_allocations) page on access.redhat.com.
```

Reply:
We're working on that in rh_cloud; users are already able to see the data that would otherwise have been sent to console, and I believe there's work going on to make that easier as well. Rich and @chris1984 may know more

Reply:
I know answers/options exist, I just don't want to point customers to a document that creates the appearance that we either don't know or don't care about what this means for disconnected users. Even for connected users who have no reason to care, it just _looks_ sloppy.

Should we keep the current link for now given that we can always change the string later before releasing? Or do we have a guarantee that someone is actively working on the other document and it will be completed in time?

Reply:
Given that we already link to this from the UI, I think it's better to link to it here as well. I'm confident that question won't remain blank for long.

Reply:
Yeah, I totally forget to actually answer that one. I'll do the needful and update

Reply:
Also, the [SCA KCS](https://access.redhat.com/articles/simple-content-access) should be accessible without login. I purpose keep that doc outside the paywall as SCA does have value for non-Satellite users (if for no other reason than they don't have to attach custom subs either). 

That said, you can also use [the official product doc](https://access.redhat.com/documentation/en-us/subscription_central/1-latest/html-single/getting_started_with_simple_content_access/index)

Reply:
I think ideally, we link something here that provides a quick summary, and that quick summary itself contains a link to the full documentation. Does that make sense on your end @sideangleside and is there a preferred link to use?

Reply:
@wbclark Lets just use the KCS. It's easier to update in place.

Reply:
Thanks for the feedback. I'll keep the link to the KCS as-is, and mark this conversation resolved.